find-darkbox.lic

```diff
@@ -15,9 +15,10 @@ arg_definitions = [
 args = parse_args(arg_definitions)
 @override = args.override
 
-junk_list = get_settings.darkbox_junk.map { |x| /#{x}/i }
+trash_items = get_settings.hollow_eve_junk.map { |x| /#{x}/i }
 darkbox_stop_on_wounded = get_settings.darkbox_stop_on_wounded
```

Comment:
💡 I'd prefer to only call `get_settings` once and cache it

Reply:
Made settings = get_settings and adjusted the rest accordingly.